src/seedu/addressbook/data/AddressBook.java

```diff
@@ -1,9 +1,6 @@
 package seedu.addressbook.data;
 
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Do not use `*` when importing. Instead, import each classes individually. (You can use IntelliJ to automate this process)